bin/cli.js

```diff
@@ -235,11 +237,11 @@ function invoke(env) {
     });
 
   commander
-    .command('migrate:down')
-    .description('        Undo the last migration performed.')
-    .action(() => {
+    .command('migrate:down [<name>]')
+    .description('        Undo the last or the defined migration performed.')
```

Comment:
`that was already run` might be more consistent with woring of `up` command

Reply:
done